src/deroff.rs

```diff
@@ -1028,17 +1044,83 @@ impl Deroffer {
         }
     }
 
-    fn do_line(&self, s: &str) -> bool {
-        match s.chars().nth(0) {
-            Some('.') | Some('\'') => !request_or_macro(s),
-            Some(c) => {
+    fn do_tbl(&mut self) -> bool {
+        match self.tblstate {
+            TblState::Options => {
+                while !self.s.is_empty() && !"\n;".contains(self.str_at(0)) {
+                    self.skip_leading_whitespace();
+
+                    if !self.str_at(0).chars().all(|c| c.is_alphabetic()) {
+                        self.skip_char(1);
+                    } else {
+                        // find first non-alphabetic character
+                        match self.s.chars().position(|c| !c.is_alphabetic()) {
+                            Some(idx) if self.s.get(idx..=idx) == Some("(") => {
+                                // self.s -> option '(' arg '(' rest
+                                let option = self.s.split_at(idx).0;
+                                let mut iter = self.s.split_at(idx + 1).1.splitn(2, '(');
```

Comment:
```suggestion
                                let option = &self.s[..idx];
                                let mut iter = self.s[idx + 1..].splitn(2, '(');
```
suggested by @scooter-dangle

Reply:
Are we sure we want to use raw indexing?

Reply:
`&string[idx..]` is logically equivalent to `string.split_at(idx).1` (and same for `..idx` case). Same panic conditions. Using `split_at` when we're only grabbing one element of the resulting tuple feels like an odd avoidance of direct indexing.

Reply:
cool, just wanted to make sure